transformer_engine/jax/csrc/extensions/ffi.h

```diff
@@ -17,6 +17,7 @@ using Result_Type = xla::ffi::Result<xla::ffi::AnyBuffer>;
 using Error_Type = xla::ffi::Error;
 using FFI = xla::ffi::Ffi;
 using FFI_Stream_Type = xla::ffi::PlatformStream<cudaStream_t>;
+constexpr auto FFI_Traits = {xla::ffi::Traits::kCmdBufferCompatible};
```

Comment:
Maybe rename FFI_Traits to FFI_cudaGraph_Traits, to make it clear which traits it is?
maybe we will use others later and it make the code a little bit more clear.